OracleGoldenGate/21c/install-deployment.sh

```diff
@@ -82,6 +82,11 @@ EOF
     else
         $(run_as_ogg) tar xf /tmp/installer/*.tar -C "${OGG_HOME}"
     fi
+    
```

Comment:
These should be in `Dockerfile`, and any permissions mismatches between the mounted scripts volume will need to be addressed in the host environment.

Reply:
I don't think I changed anything in this area other than formatting?  The extra space between the -p and the ${OGG_HOME}.

Probably just me being pedantic about the spacing... if you prefer the space, I can put it back.

Reply:
Unless reformatting with a purpose, whitespace-only changes should be avoided.

Reply:
reverted the whitespace changes